### PR TITLE
utils_misc: Avoid configuring Selinux for Ubuntu

### PR DESCRIPTION
**Pull request by balamuruhans**
Configuring selinux for ubuntu fails the test case, so it is handled to
avoid from API, selinux will remain disabled in ubuntu as it is not supported
or selinux commands will not be available. so selinux can be configured only if is 
available and enabled.

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -3820,16 +3820,18 @@ class SELinuxBoolean(object):
...
         self.server_ip = params.get("server_ip", None)
+        self.ssh_user = params.get("server_user", "root")
+        self.params = params
         if self.server_ip:
             # Setup SSH connection
             from utils_conn import SSHConnection
             self.ssh_obj = SSHConnection(params)
             ssh_timeout = int(params.get("ssh_timeout", 10))
             self.ssh_obj.conn_setup(timeout=ssh_timeout)
+            self.check_if_ubuntu_remote = self.check_if_ubuntu(remote=True)
```
Comment (will-Do) [inline virttest/utils_misc.py]: Instead hard code a new function for ubuntu in this Class, I think you may need a more general way to check if 'selinux' is supported in the place where calling the config function.

Comment (balamuruhans) [inline virttest/utils_misc.py]: Okay, I will change it accordingly

**Comment by balamuruhans** [comment]
@will-Do have made changes as per your suggestion

**Comment by balamuruhans** [comment]
@will-Do Thanks for review

**Pull request merged**